tjreports/site/views/reports/tmpl/default.php

```diff
@@ -313,7 +335,7 @@ function getQueryResult(id, Itemid)
 
 function loadReport(reportToLoad,mid)
```

Comment:
@twsvaishali make sure you are taking the service & UI approach when separating the JS files.